soql-server-pg/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+# Set everything to be logged to the file target/unit-tests.log
```

Comment:
Could we do something like use environment vars to point at a different
log4j config, or an SBT option that only gets picked up outside of
IntelliJ?  (Not sure what IntelliJ picks up or doesn't, since now it also
reads files)

On Mon, Apr 6, 2015 at 9:46 PM, Marc Slemko <notifications@github.com>
wrote:

> In soql-server-pg/src/test/resources/log4j.properties
> <https://github.com/socrata/soql-postgres-adapter/pull/53#discussion_r27851377>
> :
>
> > @@ -0,0 +1,11 @@
> > +# Set everything to be logged to the file target/unit-tests.log
>
> Hrm. We may need to figure out some different way to deal with different
> development tools here. For example, when I run tests in IntelliJ then I
> really want messages to go to stdout because IntelliJ automatically groups
> them under the right tests so I can easily see the logs associated with a
> failing test.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/socrata/soql-postgres-adapter/pull/53/files#r27851377>
> .
>



-- 
Evan Chan   |   Principal Engineer, Socrata     |    @evanfchan
*Socrata <http://www.socrata.com/> *| The Data Experience Company™
*Socrata isn't about worldly success, but about changing the world
successfully. This is why I'm excited to be here.*
*What does Socrata do? <https://www.youtube.com/watch?v=yyz86MlkriQ> *(watch
our 2 min video)